### PR TITLE
Prevent misunderstandings in acceptance-tests

### PR DESCRIPTION
**Pull request by martiniil**

### CODE REVIEW

File: pilz_robot_programming/test/acceptance_tests/acceptance_test_goal_already_reached.py [hunk 1]
```diff
@@ -140,8 +140,9 @@ def _test_repeat_circ_pose(robot):
     except RobotMoveFailed:
       pass
 
-    _askSuccess(_test_repeat_circ_pose.__name__, 'Failed to create path object for circle. '
-                                                 'Circle : Plane for motion is not properly defined.')
+    _askSuccess(_test_repeat_circ_pose.__name__, 'No INVALID_GOAL should appear from controller. The following error '
+                                                 'is o.k. Failed to create path object for circle.Circle : Plane for '
```
Comment (jschleicher) [inline pilz_robot_programming/test/acceptance_tests/acceptance_test_goal_already_reached.py]:
```suggestion
                                                 'is o.k.: Failed to create path object for circle. Circle : Plane for '
```

**Review by jschleicher — APPROVED** [review]
👍

**Comment by jschleicher** [comment]
Should this better be targeted to  noetic and backported to melodic?

**Comment by martiniil** [comment]
> 
> 
> Should this better be targeted to noetic and backported to melodic?

Yes, once #331 is through nothing speaks against it.

**Pull request merged**